acrobat/scripts/scripts.js

```diff
@@ -201,12 +99,17 @@ const CONFIG = {
   loadStyles(paths);
 
   // Import base milo features and run them
-  const {
-    loadArea, loadDelayed, loadScript, setConfig, loadLana, getMetadata, getLocale
-  } = await import(`${miloLibs}/utils/utils.js`);
+  const [
+    { locales },
+    { loadArea, loadDelayed, loadScript, setConfig, loadLana, getMetadata, getLocale }
+  ] = await Promise.all([
```

Comment:
Doing this in parallel is better, but everything that comes before loadArea is critical, so we had inlined this stuff on purpose (even though duplicating code is not the ideal approach)

What is the motivation for changing this?

@Blainegunn

Reply:
...ideally we'd have a rollup/webpack builder process our imports and do this for us.